collect_app/src/main/java/org/odk/collect/android/external/FormUriActivity.kt

```diff
@@ -46,16 +52,19 @@ class FormUriActivity : ComponentActivity() {
             val uriProjectId = uri?.getQueryParameter("projectId")
             val projectId = uriProjectId ?: firstProject.uuid
 
-            logAnalytics(uriProjectId)
-
             if (projectId == currentProjectProvider.getCurrentProject().uuid) {
-                openForm.launch(
-                    Intent(this, FormEntryActivity::class.java).also {
-                        it.action = intent.action
-                        it.data = uri
-                        intent.extras?.let { sourceExtras -> it.putExtras(sourceExtras) }
-                    },
-                )
+                if (!formFillingAlreadyStarted) {
+                    logAnalytics(uriProjectId)
```

Comment:
I don't think moving this here is correct, `logAnalytics` was being called in the case where `projectId` did not equal `currentProjectProvider.getCurrentProject().uuid` before. It might be that you just want to split out the different events in `logAnalytics` into individual calls.

Reply:
Yes but if I put it in that else block I would also need to make sure to log it only once not like now. Plus i'm wondering if we need it there... in that case we just display a dialog and that's it maybe we should have those logs only for cases where form filling is really started?

Reply:
Yeah actually, now I think about it, I'm not sure if I particularly care about those analytics events any longer. All three events have fired 1000s of times, so there's no "dead code" and I don't think we're close to being able (or needing) to deprecate form URI actions without project IDs. Can we just kill analytics here rather than reworking the code for it @lognaturel?

Reply:
@lognaturel said that we can get rid of it so I will do that.